subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/FunctionalInterfaceExtensionsTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl
+
+import org.gradle.api.Action
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import java.util.concurrent.Callable
+
+
+class FunctionalInterfaceExtensionsTest {
+
+    @Test
+    fun `Callable#call can be called using invoke`() {
+        val answer = "42"
+        val answerCallable = Callable { answer }
+
+        assertEquals(answer, answerCallable())
+    }
+
+    @Test
```

Comment:
:x: that test has nothing to do with the changes in this PR and it is already covered, please remove it